collection-scripts/gather_instancetypes

```diff
@@ -4,7 +4,19 @@ DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 source "${DIR_NAME}/common.sh"
 check_command
 
-resources=(virtualmachineinstancetype virtulmachineclusterinstancetype virtualmachinepreference virtualmachineclusterpreference)
+resources=(virtualmachineinstancetype virtualmachineclusterinstancetype virtualmachinepreference virtualmachineclusterpreference)
 
 echo "${resources[@]}" | tr ' ' '\n' | xargs -t -I{} -P "${PROS}" --max-args=1 sh -c 'echo "inspecting $1" && oc adm inspect --dest-dir ${BASE_COLLECTION_PATH} --all-namespaces $1' -- {}
 
+function gather_controller_revision() {
+  oc_project=$(echo "$1" | awk -F_ '{print $1}')
+  revision_name=$(echo "$1" | awk -F_ '{print $2}')
+  echo "inspecting ${revision_name} controller revision"
+
+  /usr/bin/oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" -n "${oc_project}" controllerrevision "${revision_name}"
+}
+
+export -f gather_controller_revision
+controller_revisions=$(oc get controllerrevision --all-namespaces | grep virtualmachine | awk '{print $1 "_" $2}')
```

Comment:
From [KubeVirt `v1.0.0`](https://github.com/kubevirt/kubevirt/blob/877bd0e3fb2396941108173a111f99db225d175a/staging/src/kubevirt.io/api/instancetype/register.go#L46-52) we have started to label all of the `instancetype.kubevirt.io` `ControllerRevisions`, IMHO I think it would be valid to only gather these labelled objects here with something like:

```
oc get controllerrevision --all-namespaces -linstancetype.kubevirt.io/object-version
```

Reply:
We are bumping CI to OCP 4.13 (https://github.com/openshift/release/pull/42972) for now, as there are issues with bumping to 4.14. Please correct me if I'm wrong, in 4.13 we don't have these labels, so this is not possible yet.

Reply:
Correct, we don't have these labels in `v4.13`.